pkg/shell/hosts.jsx

```diff
@@ -29,10 +28,12 @@ class HostsSelector extends React.Component {
     }
 
     componentDidMount() {
+        const hosts_sel = document.getElementById("nav-hosts");
         hosts_sel.appendChild(this.el);
     }
 
     componentWillUnmount() {
+        const hosts_sel = document.getElementById("nav-hosts");
```

Comment:
Yeah, but it's only necessary here because '#nav-hosts' doesn't exist yet (in this PR only) when the file is loaded. Also, I hope we can get rid of this DOM manipulation completely in this PR. Eventually.